lib/index.ts

```diff
@@ -45,10 +46,25 @@ export async function inspect(root, targetFile, options: Options = {}) {
     getMetaData(root, targetFile),
     getDependencies(root, targetFile),
   ]);
-  return {
-    plugin: result[0],
-    package: result[1],
-  };
+
+  const hasDepGraph = result[1].dependencyGraph && !result[1].dependencyTree;
```

Comment:
why it's only checking in second element?

Reply:
first element is metadata?